app/components/primer/alpha/action_list.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+module Primer
+  module Alpha
+    # An `ActionList` is a TBD
+    class ActionList < Primer::Component
+      DEFAULT_ROLE = :list
+      DEFAULT_TAG = :ul
```

Comment:
@joelhawksley do you think we need this? I don't see a scenario where this _isn't_ a `ul`/`li`